### ISSUE TITLE
New authentication concept / social login

### ISSUE THREAD

**Issue by doxsch** [yb-hackathon-code-butcher/codebase#76]
we need a better authentication concept. generating uuid in the frontend seems to me to be an unsatisfactory approach. Maybe we can use [social logins](https://www.callicoder.com/spring-boot-security-oauth2-social-login-part-1/) or offer registration to the players.

**Comment by bbortt** [comment]
yes, but I would suggest to handle this in a [UAA approach](https://www.jhipster.tech/using-uaa/). basically using `zuul`. it's webscale :D

**Comment by bbortt** [comment]
what social providers do we want to implement?
`priority:low`
